History/js/new.js

```diff
@@ -240,6 +169,80 @@ var callbacks = {
 };
 
 /**
+ * Transform the row object into a DOM object for a single message
```

Comment:
Suggestion:

`Transform the row object into a DOM object for a single message` -> `Create a node for insertion into the jquery datatables plugin using plain javascript. Avoiding jquery and datatables APIs for this task reduces the risk of the content server XSS.`

Also, please note that multiple applications get listed by the History application, not just the "Message" application.